ui/components/GrafanaComponent.js

```diff
@@ -74,7 +74,7 @@ class GrafanaComponent extends Component {
 
       grafanaURL,
       grafanaAPIKey,
-      grafanaBoardSearch: '', // we probably dont need this retrieved from store
```

Comment:
Maybe you are right but as far as I know, non assigned variables have `undefined` as value. And using `undefined` as a variable in template string causes it to have undefined in string type.

Example:
```js
var aNonAssignedVariable;
var constructedString = `prefix${aNonAssignedVariable}suffix`;
console.log(constructedString) // "prefixundefinedsuffix"
```

Reply:
We absolutely don't want this to be sent in the query parameter. Empty string is way better and can be validated in the server side.

Reply:
Done